test/dof/test_assembler.jl

```diff
@@ -210,6 +210,72 @@
         @test all((A[3, 1], A[3, 2]) .≈ (-1.0 / 6.0, 1.0 / 6.0))
     end
 
+    @testset "Poisson DG" begin
+        degree = 3
+        degree_quad = 2 * degree + 1
+        γ = degree * (degree + 1)
+        n = 4
+        Lx = 1.0
+        h = Lx / n
+
+        uref = PhysicalFunction(x -> 3 * x[1] + x[2]^2 + 2 * x[1]^3)
+        f = PhysicalFunction(x -> -2 - 12 * x[1])
+        g = uref # boundary condition
+
+        avg(u) = 0.5 * (side⁺(u) + side⁻(u))
+
+        # Build mesh
+        meshParam = (nx = n + 1, ny = n + 1, lx = Lx, ly = Lx, xc = 0.0, yc = 0.0)
+        tmp_path = "tmp.msh"
```

Comment:
tiny improvement 
`tmp_path = joinpath(mktemp(), "mesh.msh")` or `tmp_path = joinpath(tempdir(), "mesh.msh")` and then you don't have to remove the file after reading it

Reply:
It does not work on my local computer. Let's do this in a next PR.